src/game/core/explore.js

```diff
@@ -11,24 +10,27 @@ import {
 
 import { Observable } from 'rx'
 
-import models from '../models'
+import {
+  maps,
+  monsters,
+} from '../models'
+
 import { run } from './combat'
 
-export function randomMonster (mapId) {
-  const mapObj = models.maps.find(mapId)
-  const monsterPool = mapObj.monsters.reduce((acc, monster) => [
+export function randomMonster (monsters, pickMonsterFn=monsters.find) {
+  const monsterPool = monsters.reduce((acc, monster) => [
     ...acc,
-    ...Array.from({ length: monster.influence }).map(always(monster.id)),
+    ...Array.from({ length: monster.influence }, ()=> monster.id ),
```

Comment:
Understanding Ramda is a prerequisite for messing with project's core, so Ramda knowledge shouldn't be an argument. About readability, what I mean is that in plain english `always(monster)` is more expressive than `() => monster`, that's why we prefer using Ramda where possible :smile: